src/devtools/WebAudioEventObserver.js

```diff
@@ -35,27 +35,13 @@ export class WebAudioEventObserver extends Observer {
           );
         });
       };
-      const onNavigated = () => {
-        chrome.debugger.sendCommand(
-          {tabId},
-          ChromeDebuggerWebAudioDomain.Methods.enable,
-        );
-      };
 
-      chrome.debugger.attach({tabId}, debuggerVersion, () => {
-        chrome.debugger.sendCommand(
-          {tabId},
-          ChromeDebuggerWebAudioDomain.Methods.enable,
-        );
-      });
       chrome.debugger.onDetach.addListener(onDetach);
       chrome.debugger.onEvent.addListener(onEvent);
-      chrome.devtools.network.onNavigated.addListener(onNavigated);
```

Comment:
@mzgoddard: Is there a specific reason for re-enabling the debugging through `sendCommand` when `onNavigated` is triggered? Even after removing the code, Web Audio DevTools works when navigating between different pages.